variables.md

```diff
@@ -25,6 +25,14 @@ For instance:
 
 It is important to note that **all variables can be used as inputs**. This means that even if the *basic income* can be calculated from other variables, I can, for a given simulation, provide it as an input. Then, if another formula asks for the value of *basic income* for a month, the input value will be returned, and the *basic income* formula **won't be executed.**
 
+### Default values
 
- 
+When OpenFisca is not able to calculate the value of a variable for a requested period, a **default value** is returned.
 
+The default value of a variable is returned:
+  - When the value of an input variables is requested, if this variable has not been set in the input for the requested period.
```

Comment:
>"When a user requests the value of an input variable and they have not provided it for the requested period"